style.css

```diff
@@ -1,100 +1,376 @@
-@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&display=swap");
-
-.container {
-  display: flex;
-  justify-content: space-between;
-}
-
-.menu {
-  display: flex;
-  border: 8px;
-  margin: 10px;
-  padding: 8px;
-  align-items: center;
-  width: 40px;
-  height: 40px;
-  justify-content: center;
-}
-
-.logo-placeholder {
-  display: flex;
-  padding: 10px 12px;
-  align-items: center;
-  justify-content: center;
-  border: 8px;
-  margin: 10px;
-}
-
-.my-logo {
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 18px;
-  line-height: 20px;
-  color: #6070ff;
-  text-decoration: none;
-}
-
-.container-headline {
-  display: flex;
-  justify-content: space-between;
-  align-items: flex-start;
-  background-image: url("images/header.svg");
-  background-repeat: repeat;
-}
-
-.headline {
-  display: flex;
-  height: 300px;
-  padding: 50px 24px;
-  flex-wrap: wrap;
-}
-
-.primary-text {
-  display: flex;
-  align-items: flex-end;
-  justify-content: center;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  color: #172b4d;
-  font-size: 40px;
-  line-height: 52px;
-}
-
-.description {
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 16px;
-  line-height: 24px;
-  color: #344563;
-  align-items: flex-start;
-}
-
-.social-media {
-  height: 25%;
-  width: 50%;
-  align-items: flex-start;
-}
-
-.media-icons {
-  margin-top: 12px;
-  list-style: none;
-  display: flex;
-  padding: 0;
-  justify-content: space-between;
-  align-items: flex-start;
-}
-
-#media-text {
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 500;
-  font-size: 16px;
-  color: #7f8cff;
-}
-
-.icon-list {
-  display: inline-flex;
-}
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&display=swap");
+
+.container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.menu {
+  display: flex;
+  border: 8px;
+  margin: 10px;
+  padding: 8px;
+  align-items: center;
+  width: 40px;
+  height: 40px;
+  justify-content: center;
+}
+
+.logo-placeholder {
+  display: flex;
+  padding: 10px 12px;
+  align-items: center;
+  justify-content: center;
+  border: 8px;
+  margin: 10px;
+}
+
+.my-logo {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+  text-decoration: none;
+}
+
+.container-headline {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+  background-image: url("images/header.svg");
+  background-repeat: repeat;
+}
+
+.headline {
+  display: flex;
+  height: 300px;

```

Comment:
- Let's remove this height, as its breaking the design and not allowing the headline section to have it complete height.